YAXLib/Caching/MruCache.cs

```diff
@@ -0,0 +1,204 @@
+﻿// 
```

Comment:
Although a nice solution, for YAXLib the implemented FIFO cache will do (and is faster)?

Reply:
Less (code) is more, so vote for remove